pages/guide/nested.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: guide
+section: guide
+role: page
+order: 20
+toc: true
+title: Nested Suites
+label: Nested Suites
+description: |
+  Nested Suites
+---
+
+{:.toc}
+## Nested Suites 
+
+Just like your favourite test framework, `Testdeck` supports nested suites.
+
+While in your favourite test framework you might be using nested `describe`, when using `Testdeck`
+you should use `suite` instead. And while you can still use `describe`, you should use `suite`, only.
+The rationale here being that we may add support for test frameworks that do not support `describe`.
+
+{% highlight TypeScript lineos %}
+import { suite, test } from '@testdeck/mocha';
+
+describe('a suite', function() {
+
+  @suite
+  class NestedSuite {
+
+    @test
+    failingTest() {
+
+      throw new Error('not implemented yet');
+    }
+  }
+});
+{% endhighlight %}
+
+Let's replace the `describe` by `suite` and add an extra level of nesting.
+
+{% highlight TypeScript lineos %}
+import { suite, test } from '@testdeck/jasmine';
+
+suite('a suite', function() {
```

Comment:
Just a note - I am not sure if these are currently implemented, but they are in the technical dept and must work for v1.

Reply:
I found that you are using this in the tests for core or some seed.